bin/test_staging_update_census_mapbox

```diff
@@ -22,72 +31,73 @@ def make_location(lat, long)
 end
 
 def main
```

Comment:
Hide whitespace to see the actual changes other than indenting from this `if` statement

Reply:
TIL hide whitespace!! 🙌